http/oidc/src/main/java/org/wildfly/security/http/oidc/OidcRequestAuthenticator.java

```diff
@@ -535,7 +550,7 @@ private void addScopes(String scopes, Set<String> allScopes) {
         }
     }
 
-    private String convertToRequestParameter(URIBuilder redirectUriBuilder, String redirectUri, String state, List<NameValuePair> forwardedQueryParams) throws JoseException, IOException {
+    private String convertToRequestParameter(URIBuilder redirectUriBuilder, String redirectUri, String state, List<NameValuePair> forwardedQueryParams, String cryptoValue) throws JoseException, IOException {
```

Comment:
Same comment here about cryptoValue, would be good to rename.

Reply:
done